runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,133 @@
 
 /* @ddr_namespace: map_to_type=J9JavaAccessFlags */
 
-/* Constants from J9JavaAccessFlags */
-#define J9AccAbstract 0x400
-#define J9AccAnnotation 0x2000
-#define J9AccBridge 0x40
+/*
+ * Modifiers of ROM classes, fields or methods.
+ * When a given modifier is defined by the JVM specification,
+ * the value is taken from that specification.
+ *
+ * See map in ROMClassWriter::writeMethods to confirm available slots for methods.
+ */
+#define J9AccPublic                        0x00000001 /* class method field */
+#define J9AccPrivate                       0x00000002 /* class(nested) method field */
+#define J9AccProtected                     0x00000004 /* class(nested) method field */
+#define J9AccStatic                        0x00000008 /* class(nested) method field */
+#define J9AccFinal                         0x00000010 /* class method field */
+/* Project Valhalla repurposes 0x000000020 as ACC_IDENTITY since ACC_SUPER has no effect after Java 8. */
+#define J9AccSuper                         0x00000020 /* class */
+#define J9AccClassHasIdentity              0x00000020 /* class */
+#define J9AccSynchronized                  0x00000020 /* method */
+#define J9AccValueType                     0x00000040 /* class(Valhalla) */
+#define J9AccBridge                        0x00000040 /* method */
+#define J9AccVolatile                      0x00000040 /* field */
+#define J9AccVarArgs                       0x00000080 /* method */
+#define J9AccTransient                     0x00000080 /* field */
+#define J9AccNative                        0x00000100 /* method */
+#define J9AccInterface                     0x00000200 /* class */
+#define J9AccGetterMethod                  0x00000200 /* method */
+#define J9AccAbstract                      0x00000400 /* class method */
+#define J9AccPrimitiveValueType            0x00000800 /* class(Valhalla) */
+#define J9AccStrict                        0x00000800 /* method */
+#define J9AccSynthetic                     0x00001000 /* class method field */
+#define J9AccAnnotation                    0x00002000 /* class */
+#define J9AccForwarderMethod               0x00002000 /* method */
+#define J9AccEnum                          0x00004000 /* class field */
+#define J9AccEmptyMethod                   0x00004000 /* method */
+/* ACC_MODULE reserves 0x8000 for classes*/
```

Comment:
Please insert a space before `*/`.

Reply:
As well as the missing space, perhaps `0x8000` should be `0x00008000` for consistency.